event-management-frontend/src/components/eventCreateOrEdit/ticketsSection/CategoryCard/CategoryCardDumb.tsx

```diff
@@ -0,0 +1,221 @@
+  const [dialogDescription, setDialogDescription] = useState("");
+
+  const { t } = useTranslation();
+  const classes = useStylesCategoryCard();
+
+  let categoryData = props.ticketData.filter((data) => data.title === props.title)[0];
+  let index = props.event.ticketCategoryDtoList.findIndex((card) => card.id === props.id);
+
+  const removeThisCard = () => {
+    if (categoryData && categoryData.sold !== 0) {
+      setDisableMessage("Disable");
+      setDialogTitle(t("categoryCard.removeTitle"));
+      setDialogDescription(t("categoryCard.removePurchased"));
+      setOpen(true);
+    } else if (
+      props.title.trim() !== "" ||
+      props.subtitle.trim() !== "" ||
+      props.price !== 0 ||
+      props.description.trim() !== "" ||
+      props.ticketsPerCategory !== 0
+    ) {
+      setDisableMessage("");
+      setDialogTitle(t("categoryCard.removeTitle"));
+      setDialogDescription(t("categoryCard.removeEmpty"));
+      setOpen(true);
+    } else {
+      removalApproved();
+    }
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const removalApproved = () => {
+    props.removeCard(props.id);
+    console.log("torles: ", props.event.ticketCategoryToDelete);
+  };
+
+  return (
+    <div>
+      <Grid item xl={12} lg={12} md={10} sm={12} xs={10}>
+        <Card variant="outlined" className={`${classes.root} `}>
+          <CardContent className={classes.cardStyle}>
+            <Grid item xl={4} sm={7} xs={11} className={classes.marginBasic2}>
+              <TextField
+                required
+                className={classes.marginBasic}
+                InputProps={{
+                  className: classes.inputBasic,
+                }}
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                name="title"
+                variant="outlined"
+                label={t("categoryCard.title")}
+                error={props.formErrors.ticketCategoryDtoList[index].title.length > 0}
+                helperText={props.formErrors.ticketCategoryDtoList[index].title}
+                defaultValue={props.title}
+                onChange={props.handleChange}
+              />
+            </Grid>
+            <Grid item xl={4} sm={7} xs={11} className={classes.marginBasic2} justify="center" alignItems="center">
+              <TextField
+                variant="outlined"
+                className={`${classes.marginBasic}`}
+                InputProps={{
+                  className: classes.inputBasic,
+                }}
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                name="subtitle"
+                label={t("categoryCard.subtitle")}
+                defaultValue={props.subtitle}
+                onChange={props.handleChange}
+                error={props.formErrors.ticketCategoryDtoList[index].subtitle.length > 0}
+                helperText={props.formErrors.ticketCategoryDtoList[index].subtitle}
```

Comment:
Poti folosi "Destructuring Props" pentru a nu repeta "props." (si a simplifica bucatelele de cod:D)
Observatie valabila in tot fisierul